modules/_labs/header/react/lib/parts/SearchBar.tsx

```diff
@@ -195,61 +213,63 @@ const CloseButton = styled(IconButton)<
 
   return {
     position: `absolute`,
-    top: spacing.zero,
-    right: spacing.zero,
-    bottom: spacing.zero,
+    top: 0,
+    bottom: 0,
+    right: 0,
     margin: `auto ${spacing.xxs}`,
     zIndex: 3,
     ...collapseStyles,
   };
 });
 
 const SearchField = styled(FormField)<
-  Pick<SearchBarProps, 'isCollapsed' | 'grow'> & Pick<SearchBarState, 'showForm'>
->(({isCollapsed, showForm, grow}) => {
+  Pick<SearchBarProps, 'isCollapsed' | 'grow' | 'height'> & Pick<SearchBarState, 'showForm'>
+>(({isCollapsed, showForm, grow, height}) => {
   return {
     display: (isCollapsed && showForm) || !isCollapsed ? 'inline-block' : 'none',
     width: '100%',
     height: height,
     maxWidth: isCollapsed || grow ? '100%' : maxWidth,
-    marginBottom: spacingNumbers.zero,
+    marginBottom: spacing.zero,
     '> div': {
       display: 'block',
     },
   };
 });
 
 const SearchInput = styled(TextInput)<
-  Pick<SearchBarProps, 'isCollapsed' | 'grow'> & {inputColors: ReturnType<typeof getInputColors>}
->(({isCollapsed, inputColors, grow}) => {
+  Pick<SearchBarProps, 'isCollapsed' | 'grow' | 'height'> & {
+    inputColors: ReturnType<typeof getInputColors>;
+  }
+>(({isCollapsed, inputColors, grow, height}) => {
   const collapseStyles: CSSObject = isCollapsed
     ? {
-        ...type.h3,
-        fontWeight: 400,
+        fontSize: '20px',
+        lineHeight: '20px', // For ie11, line-height needs to match font-size
```

Comment:
We were using `type.h3` _just_ to get the font-size. Anything else that `type.h3` added we just overwrote below. Probably not a good idea. I'm explicitly spelling out the size we want here.

I could have done `type.h3.fontSize` I suppose but it's a sure way to get a JavaScript error if that ever changes?

Reply:
Fine in this case since it's a one-off component.